pkg/email/send_email.go

```diff
@@ -0,0 +1,38 @@
+package email
+
+import (
+	"gopkg.in/gomail.v2"
+)
+
+type Emailer struct {
+	SMTPHost string
+	Port     int
+	Username string
+	Password string
+	Sender   string
+}
+
+type EmailParams struct {
+	SMTPHost string
+	Port     int
+	Username string
+	Password string
+	Sender   string
+}
```

Comment:
Why do we need two structs representing the exact same thing and the NewEmail is only passing parameters between the two structs?

Reply:
One holds config email params and one holds the email functionality. Its kind of weird a config struct should hold functionality on it.